cartographer/sensor/compressed_point_cloud.cc

```diff
@@ -158,16 +149,15 @@ void CompressedPointCloud::ConstIterator::ReadNextPoint() {
 }
 
 CompressedPointCloud::CompressedPointCloud(const PointCloud& point_cloud)
-    : point_data_(Compress(point_cloud)), num_points_(point_cloud.size()) {}
+    : point_data_(DoCompress(point_cloud)), num_points_(point_cloud.size()) {}
 
 CompressedPointCloud::CompressedPointCloud(const std::vector<int32>& point_data,
                                            size_t num_points)
     : point_data_(point_data), num_points_(num_points) {}
 
-CompressedPointCloud CompressedPointCloud::CompressAndReturnOrder(
-    const PointCloud& point_cloud, std::vector<int>* new_to_old) {
-  return CompressedPointCloud(Compress(point_cloud, new_to_old),
-                              point_cloud.size());
+CompressedPointCloud CompressedPointCloud::Compress(
```

Comment:
Remove completely since this is now the same as calling the constructor directly?